.github/workflows/main.yml

```diff
@@ -130,5 +130,54 @@ jobs:
         with:
          name: app reports
          path: app/build/test-results
-      
-      
+
+  bazel_tests:
+    name: Bazel tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        id: cache
+        with:
+          path: ~/oppia-bazel/bazel-bin/src/bazel
+          key: ${{ runner.os }}-bazel-${{ hashFiles('**/*.bazel') }}}
+      - name: Setup bazel
+        uses: jwlawson/actions-setup-bazel@v1
+        with:
+          bazel-version: '3.4.1'
+      - name: Use bazel
+        run: bazel --version
+      - name: Clone Oppia Bazel
+        run: git clone https://github.com/miaboloix/bazel.git ~/oppia-bazel
+      - name: Set up JDK 9
+        uses: actions/setup-java@v1
+        with:
+          java-version: 9
+      - name: Create tmp/android_tools
+        run: |
+          mkdir tmp
+          cd tmp
+          mkdir android_tools
+      - name: Extract Android ToolsCurr
+        run: |
+          cd ~/oppia-bazel
+          unzip bazel-tools.zip
+          tar -xf ~/oppia-bazel/android_tools.tar.gz -C /home/runner/work/oppia-android/oppia-android/tmp/android_tools
```

Comment:
Here & elsewhere: can we use relative home paths instead of absolute ones in case GitHub changes their pathing setup?